jmx/src/main/java/org/jboss/as/jmx/logging/JmxLogger.java

```diff
@@ -445,7 +447,7 @@ public interface JmxLogger extends BasicLogger {
     String cannotUnregisterMBeansUnderReservedDomain(String domain);
 
     @Message(id = 46, value="The ObjectName coming from MBeanRegistration.preRegister() '%s' is in a reserved JMX domain")
-    RuntimeOperationsException badDomainInCalclulatedObjectNameException(ObjectName name);
+    RuntimeOperationsException badDomainInCalculatedObjectNameException(ObjectName name, @Param RuntimeException e);
```

Comment:
All the other instantiation of RuntimeOperationsException in this module uses the logger to create a RuntimeException of some kind and then the caller passes that into the RuntimeOperationsException constructor.  I think the same should be done here.

@kabir Any reason that wasn't done here?

Reply:
@bstansberry it is probably an oversight. It looks like if this method was ever called it would have caused an error due to the missing constructor parameter.

Looking, a better fix would probably be to have this method return an IllegalArgumentException (or similar), and then to have the caller `throw new RuntimeOperationsException(JmxLogger.ROOT_LOGGER.badDomainInCalclulatedObjectNameException(registeredName))` (i.e. what you said :-) )

Reply:
updated 😃👍🏻